main/test/ca/uwaterloo/flix/language/phase/TestWeeder.scala

```diff
@@ -1051,18 +1051,6 @@ class TestWeeder extends AnyFunSuite with TestUtils {
     expectError[WeederError.InlineAndDontInline](result)
   }
 
-  test("MalformedFloat64.01") {
-    val input = "def f(): Float64 = 1.7976931348623158e+308"
```

Comment:
Previously these were lexed as '1.7976931348623158e', '+', and '308' which caused the malformed error. But this is a valid float so im not sure why this test was added